plugin/tower/pkg/controller/endpoint/controller.go

```diff
@@ -58,16 +60,33 @@ type Controller struct {
 	endpointLister         informer.Lister
 	endpointInformerSynced cache.InformerSynced
 
+	systemEndpointInformer       cache.SharedIndexInformer
+	systemEndpointLister         informer.Lister
+	systemEndpointInformerSynced cache.InformerSynced
+
+	everouteClusterInformer       cache.SharedIndexInformer
+	everouteClusterLister         informer.Lister
+	everouteClusterInformerSynced cache.InformerSynced
+
 	// endpointQueue contains endpoint to process. The element in queue
 	// is endpoint name. And we use vnic ID as endpoint name.
 	endpointQueue workqueue.RateLimitingInterface
+
+	staticEndpointQueue workqueue.RateLimitingInterface
 }
 
 const (
 	vnicIndex = "vnicIndex"
 	vmIndex   = "vmIndex"
 
-	ExternalIDName = "iface-id"
+	ExternalIDName         = "iface-id"
+	DefaultExternalIDValue = "default-externalID-value"
+
+	DynamicEndpointPrefix    = "tower.ep.dynamic"
+	StaticEndpointPrefix     = "tower.ep.static"
```

Comment:
add a '-' in the end?

Reply:
These two mean TYPE, 	actually used are:
ControllerEndpointPrefix = "tower.ep.static.ctrl-"
SystemEndpointPrefix     = "tower.ep.static.sys-"

Reply:
the "StaticEndpointPrefix" will be used in "getCtrlEndpointName", and the name will become as "tower.ep.static**xxxxx**-ip"